src/internal/git/fetch.go

```diff
@@ -8,8 +8,8 @@ import (
 	goConfig "github.com/go-git/go-git/v5/config"
 )
 
-// fetchTag performs a `git fetch` of _only_ the provided tag
-func fetchTag(gitDirectory string, tag string) {
+// FetchTag performs a `git fetch` of _only_ the provided tag.
+func FetchTag(gitDirectory string, tag string) {
```

Comment:
This function was made public but nothing outside of this package is calling it?